config/locales/en.yml

```diff
@@ -203,7 +203,7 @@ en:
         sector: Sector
         status: Status
         tied_status: Tied status
-    dashboard: Dashboard
+    dashboard: Home
```

Comment:
Should we change this to something else as we have removed the concept of a 'dashboard' entirely? Maybe something like `home_navigation_item` or just `home`?